src/pages/public_registration/log/LogEntry.tsx

```diff
@@ -139,3 +144,50 @@ const LogHeaderIcon = ({ topic }: Pick<LogEntryObject, 'topic'>) => {
       return null;
   }
 };
+
+const getfileLogEntryTitle = (logEntry: LogEntryObject, t: TFunction) => {
+  switch (logEntry.topic) {
+    case 'PublicationCreated':
+      return t('log.titles.result_created');
+    case 'PublicationPublished':
+    case 'PublicationImported':
+      return t('log.titles.result_published');
+    case 'PublicationRepublished':
+      return t('log.titles.result_republished');
+    case 'PublicationUnpublished':
+      return t('log.titles.result_unpublished');
+    case 'PublicationDeleted':
+      return t('log.titles.result_deleted');
+    case 'DoiReserved':
+      return t('log.titles.doi_reserved');
+    case 'DoiRequested':
+      return t('log.titles.doi_requested');
+    case 'DoiRejected':
+      return t('log.titles.doi_rejected');
+    case 'DoiAssigned':
+      return t('log.titles.doi_given');
+    case 'FileUploaded':
+      return t('log.titles.files_uploaded', { count: 1 });
```

Comment:
Denne brukes i gamle loggen også, så har tilpasset denne til å bruke `count: 1` i stedet for å legge inn en ny nøkkel. Det blir trolig overflødig når vi får tatt vekk dagens logg